README.md

```diff
@@ -12,9 +12,9 @@ stakeholders can be confident about the direction the project is evolving in.
 
 ## How to submit an RFC
 
-1. Copy `template.md` into `text/YYYY-MM-DD-my-feature.md`.
+1. Copy `template.md` into `text/0000-my-feature.md`.
 2. Write the document and fill in the blanks.
-3. Submit a pull request.
+3. Submit a pull request. Then you can change the `0000` in your filename to your pull request ID in another commit.
```

Comment:
I think this can be a little clearer. In particular the word "can" is too weak. This step is also two steps in one. I was also unsure about the term "pull request __ID__", and from glancing through the GitHub API docs I think the canonical term is "pull request __number__", which I also think is clearer than "ID".

This is also the only indication in the document that RFC numbers are identical to PR ids, but it does not say so explicitly - it says that filenames contain PR Ids. (This point is nitpicky, but in an ideal document there would be a clear statement like "TiKV RFCs are identified by number, and that number is the same as the pull request number for the submitted RFC." That's a bigger change than I would ask for on this PR though.

So I would suggest changing this to instead say more like

`
4. In a new commit, change the `0000` in your filename to be the same as your pull request number, with leading zeros to 4 digits. This is your RFC number.
5. Push the new commit to the same branch to update the pull request with the correct RFC number.
`